pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -560,3 +597,25 @@ func (r *ReconcileCincinnati) ensureConfigMap(ctx context.Context, reqLogger log
 
 	return nil
 }
+
+func (r *ReconcileCincinnati) ensureSecret(ctx context.Context, reqLogger logr.Logger, cm *corev1.Secret) error {
```

Comment:
`cm` was an abbreviation of `ConfigMap` in the `ensureConfigMap` function, but perhaps here we should use something else. `secret` may be sufficient.